src/System.Numerics.Tensors/Directory.Build.props

```diff
@@ -1,8 +1,8 @@
 ﻿<Project>
   <Import Project="..\Directory.Build.props" />
   <PropertyGroup>
-    <AssemblyVersion>0.2.0.0</AssemblyVersion>
-    <PackageVersion>0.2.0</PackageVersion>
+    <PackageVersion>5.0.0</PackageVersion>
```

Comment:
@tannergooding @ericstj should this stay at 0.2.0?

Reply:
0.3